lib/v1/gadgets/blogger.md

```diff
@@ -0,0 +1,12 @@
+gratipay.xml is a an example of Gadget for Blogger. To add it, go to
+you blog overview, then:
+
+    Layout -> Add a Gadget -> Add your own
+
+Enter URL ('https://' prefix didn't work for me):
+
+    grtp.co/v1/gadgets/blogger.xml
```

Comment:
There are not many places [where Google Gadgets are helpful](https://developers.google.com/gadgets/docs/overview), so I renamed it blogger/gratipay.xml and changed doc to README.md, so that people can read how to install it from subpage.